admin_guide/managing_pods.adoc

```diff
@@ -146,16 +146,17 @@ endif::openshift-enterprise,openshift-origin[]
 
 [[admin-guide-controlling-egress-traffic]]
 == Controlling Egress Traffic
-As an {product-title} cluster administrator, you can control egress traffic
-using either a xref:admin-guide-deploying-an-egress-router-service[router] or
-xref:admin-guide-limit-pod-access-egress[firewall] methods.
+As an {product-title} cluster administrator, you can control egress traffic in two different ways:
+
+* Using an egress xref:admin-guide-deploying-an-egress-router-service[router], so that an external source treats traffic as though it were coming from a known source.
+* Using an egress xref:admin-guide-limit-pod-access-egress[firewall], where traffic is accepted, enforced, and defined.
```

Comment:
I think this is trying to say, what https://docs.openshift.com/container-platform/3.3/admin_guide/managing_pods.html#admin-guide-limit-pod-access-egress says, but it does not quite align. 

In short `accepted, enforced, and defined` (are not how I would phrase this). 

In stead I would say: 

`Using an egress xref:admin-guide-limit-pod-access-egress[firewall] allows a cluster administrator to enforce acceptable (outbound) traffic policies. These policies can be defined in such a way that, acceptable endpoints or ip ranges (subnets) are the only acceptable targets for the dynamic endpoints (pods with in OpenShift) to talk to.`